ci/README.md

```diff
@@ -0,0 +1,85 @@
+# Integrating with Gitlab CI
+
+(These instructions are specifically for the Liip Gitlab CI, but may be used on other Gitlab CI's as well)
+
+To make it easy to run tests on the Gitlab CI runners, this packages provides some general purpose scripts, so you don't have to reinvent the wheel all the time.
+
+## Setup
+
+### Using the gitlabci role
+
+The automatic way. Just uncomment/add the folloing line in your `playbook.yml` and the needed files will be created automatically on the next provsioning (if they don't exist already)
+
+````
+    - { role: gitlabci }
+````
+
+It installs the following files (which should be added to git afterwards)
+
+#### .gitlab-ci.yml
+
+The config file for Gitlab CI, it tells the CI what exactly to run
+
+#### scripts/gitlabci.sh
+
+The script called first by the gitlab runner. It updates the submodules and then calls `./virtualization/rawbot-virtualization/ci/start.sh`, which does start vagrant, provisions it and calls your actual test script. 
+
+#### scripts/run_tests.sh
+
+This is where your actually test calls go. This is run within your vagrant box.
+
+#### virtualization/provisionbuild.dat
+
+To prevent provisioning on the ci runners all the time and save lots of time, provisioning is only run, when this file changes. Therefore if you change something in your provisioning scripts, also changes this file to a different value (doesn't matter which one, as long as it's different, but some kind of timestamp assures that it's different)
+
+If you don't add this file to your project, then provisioning will be run every time a ci build is started.
+
+#### Using a different folder than scripts/
+
+If you prefer to install those files in a different folder than scripts/ you can add the following line in your `parameters.yml`, eg:
+
+```
+ci_scripts_folder: bin/
+```
+
+You can also adjust the files afterwards and uncomment the gitlabci role in `playbook.yml` again (otherwise the files will be created again after each provisioning)
```

Comment:
That I get, but from what I understand from the provisioning, since there is `force=no` ansible won't overwrite the files if they already exists, so it should be ok.

From the doc : "the default is yes, which will replace the remote file when contents are different than the source. If no, the file will only be transferred if the destination does not exist."

So the file will be copied the first time around, and upon next provisioning, since the file will already exists, ansible won't replace the existing file.